dotCMS/src/integration-test/java/com/dotmarketing/util/contentlet/pagination/ContentletsPaginatedIntegrationTest.java

```diff
@@ -0,0 +1,332 @@
+package com.dotmarketing.util.contentlet.pagination;
```

Comment:
This test needs to be included in MainSuite.java

Reply:
done https://github.com/dotCMS/core/pull/22733/commits/e898cbc868542144ffb120d929883cb21f9e754b#diff-8c127e4dadcd12d480a62cfc2a0dd406dbcb61bab2081608e05dfd4be0df5499R564